app/src/main/java/org/stratoemu/strato/utils/CacheManagementUtils.kt

```diff
@@ -27,10 +24,9 @@ class CacheManagementUtils {
         /**
          * Deletes the cache files for a given game.
          */
-        fun deleteCacheFile(titleId : String?) : Boolean {
-            if (titleId == null) return false
+        fun deleteCacheFile(titleId : String) : Boolean {
             return File("${cacheFolderRoot}$titleId").delete()
                     && File("${cacheFolderRoot}$titleId.staging").delete()
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
```suggestion
}

```
Add empty new line at the end like this